apps/cassandra/deployers/test_vars.yml

```diff
@@ -1,6 +1,6 @@
-cassandra_deployment: cassandra_statefulset.yml
+application_deployment: cassandra_statefulset.yml
 app_ns: "{{ lookup('env','APP_NAMESPACE') }}"
 app_label: "{{ lookup('env','APP_LABEL') }}"
 test_name: cassandra-deployment
 app_replica: "{{ lookup('env','APP_REPLICA') }}"
-
+application_name: Cassandra statefulset
```

Comment:
is this needed

Reply:
Yes, Sir using this ```application_name``` to describe the task in ```/common/utils```

Reply:
Cool, nice touch wrt describing the tasks better. Sorry to nitpick - but somehow the idea of an unquoted string with spaces in the yaml bothers me. It is ok now that it is being used in `name`, but standardising it may help it's reuse somewhere else, if needed?

Reply:
I will resolve that unquoted strings